targets/TARGET_ublox/TARGET_R5030/TARGET_R5XXX/PinNames.h

```diff
@@ -55,8 +55,17 @@ typedef enum {
     UART1_CTS = p12,
     UART1_RTS = p11,
 
-    I2C_SCL = p0,
-    I2C_SDA = p1,
+    I2C1_SCL = p0,
+    I2C1_SDA = p1,
+
+	I2C_SCL = I2C1_SCL,
```

Comment:
Please replace tabs with spaces.

Reply:
I have made changes in commit "code clean-up"